libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -123,7 +124,7 @@ class AppPrompt {
       <div class="appPrompt-icon">
         ${this.image}
       </div>
-      <div class="appPrompt-title">${this.title}</div>
+      <h2 class="appPrompt-title">${this.title}</h2>
```

Comment:
Did we get SEO requirements/approval for this? I remember they wanted to keep heading element numbers low

Reply:
Yes, it's stated in the figma specs

Reply:
The Figma specs are suggesting the look of the element, not necessarily the markup to use. business.adobe.com has 2 `h2` elements, adobe.com has just one, we should probably aim to keep these numbers low. We might end up in a situation where PEP content appears in search results, when it's actually a feature for a limited set of users.

We had heading elements in the header and footer navigations and the SEO team asked us to remove them because they were polluting search results.

Reply:
I agree with Rares here, it would be good to get some SEO input on this.